core/debug/inc/debug.h

```diff
@@ -18,8 +18,19 @@
 #define __DEBUG_H__
 
 #include <uvisor.h>
+#include "debug_exports.h"
 #include "halt.h"
 
+/* Debug box handle.
+ * This is used internally by uVisor to keep track of the registered debug box
+ * and to access its driver when needed. */
+typedef struct TDebugBox {
+    const TUvisorDebugDriver *driver;
+    uint8_t box_id;
```

Comment:
Moved `box_id` to the end. I'd still keep it as `uint8_t` as that's what we use for box IDs.